community/xmlsec1/build

```diff
@@ -1,12 +1,33 @@
 #!/bin/sh -e
 
 ./configure \
-    --prefix=/usr \
-    --with-default-crypto=openssl \
-    --without-gcrypt \
-    --without-gnutls \
-    --without-nss \
-    --without-libxslt
+  --prefix=/usr \
+  --bindir=/usr/bin \
+  --sbindir=/usr/sbin \
+  --enable-manpages-build=yes \
+  --enable-mscrypto=yes \
+  --enable-mscng=yes \
+  --enable-unicode=yes \
+  --enable-md5=yes \
+  --enable-ripemd160=yes \
+  --enable-sha1=yes \
+  --enable-sha224=yes \
+  --enable-sha256=yes \
+  --enable-sha384=yes \
+  --enable-sha512=yes \
+  --enable-hmac=yes \
+  --enable-dsa=yes \
+  --enable-rsa=yes \
+  --enable-x509=yes \
+  --enable-des=yes \
+  --enable-aes=yes \
+  --enable-gost=yes \
+  --enable-xmldsig=yes \
+  --enable-xmlenc=yes \
+  --enable-soap=yes \
+  --enable-mans=yes \
+  --enable-docs=no \
+  --enable-apps=yes
```

Comment:
Do... Do these have to be explicit? Seems like most of this should be covered by `--with-default-crypto=openssl`

Reply:
To be honest, i don't have a clue, i just copied from ./configure --help in hope to enable all functionality, but i have updated the package and erased all of that, lets just trust the defaults, i don't see anything obvious missing.